docs/analyzer/user_guide.md

```diff
@@ -219,6 +222,16 @@ analyzer arguments:
                         clang-tidy checkers. The file can be dumped by
                         'CodeChecker analyzers --dump-config clang-tidy'
                         command.
+  --analyzer-config [ANALYZER_CONFIG [ANALYZER_CONFIG ...]]
+                        Analyzer configuration options in the following format:
+                        analyzer:key=value. The collection of the options can
+                        be printed with 'CodeChecker analyzers
+                        --analyzer-config'.
+  --checker-config [CHECKER_CONFIG [CHECKER_CONFIG ...]]
```

Comment:
You marked this as resolved but it still does not work for me.

Reply:
It works for me. Am I missing something?

Reply:
If I run the `CodeChecker checkers --checker-config` command it will print the checker configs:
```
 clangsa:alpha.clone.CloneChecker:MinimumCloneComplexity
 clangsa:alpha.clone.CloneChecker:ReportNormalClones
 clangsa:alpha.clone.CloneChecker:IgnoredFilesPattern
 clangsa:alpha.security.MmapWriteExec:MmapProtExec
 clangsa:alpha.security.MmapWriteExec:MmapProtRead
 clangsa:cplusplus.Move:WarnOn
 ...
```
Which is good, but if I run the `CodeChecker checkers --detail` command it will not print these options below each checkers:
```sh
 - alpha.clone.CloneChecker                                       clangsa    LOW         (Enable only for development!) Reports similar pieces of code.
 - alpha.core.BoolAssignment                                      clangsa    LOW         (Enable only for development!) Warn about assigning non-{0,1} values to Boolean variables
```

The expected output of this command would be the following:
```
- alpha.clone.CloneChecker                                       clangsa    LOW         (Enable only for development!) Reports similar pieces of code.
  Options:
  clangsa:alpha.clone.CloneChecker:MinimumCloneComplexity
  clangsa:alpha.clone.CloneChecker:ReportNormalClones
  clangsa:alpha.clone.CloneChecker:IgnoredFilesPattern
```